### ISSUE TITLE
restic mount fails on FreeBSD 13.1

### ISSUE THREAD

**Issue by GilGalaad** [restic/restic#4043]
Output of `restic version`
--------------------------
`restic 0.14.0 compiled with go1.19.2 on freebsd/amd64`

How did you run restic exactly?
-------------------------------
`restic mount /mnt/remote/`

parameters are injected via a small python wrapper that create a proper environment for the process with the following values
```
RESTIC_REPOSITORY = "s3:https://myserver:9000/restic"
AWS_ACCESS_KEY_ID = remote_key
AWS_SECRET_ACCESS_KEY = remote_secret
RESTIC_PASSWORD = passphrase
RESTIC_CACHE_DIR = /store/maintenance/restic/cache
RESTIC_PACK_SIZE = "128"
```

output is the following:
```
repository 70565c28 opened (repository version 1) successfully, password is correct
Now serving the repository at /mnt/remote/
Use another terminal or tool to browse the contents of this folder.
When finished, quit with Ctrl-c here or umount the mountpoint.
resource temporarily unavailable
```
and the process exits unexpectedly.

What backend/server/service did you use to store the repository?
----------------------------------------------------------------
minio, deployed on a remote physical server.

Expected behavior
-----------------
the mountpoint should show the content of repository

Actual behavior
---------------
the command fails without explaining what is the problem.
adding `--verbose=3` does not change the output and does not provide other informations.
on the same platform and reposity i regularily use commands `backup`, `restore`, `forget `and `prune`, and they work like a charm

Steps to reproduce the behavior
-------------------------------
on FreeBSD 13.1-RELEASE-p4 run:
```
pkg install restic
export RESTIC_REPOSITORY ...
restic init
restic mount /mnt
```

Do you have any idea what may have caused this?
-----------------------------------------------
I don't know.
I used the mount command succesfully many times in the past, but i can't remember what was the latest working combo of OS/restic version that worked.
The only thing that comes to mind is that rclone had recently issues with a change in FreeBSD kernel fuse module, but seems they are fixed now. This could be related.

Do you have an idea how to solve the issue?
-------------------------------------------
I wish I had.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
restic is an amazing piece of software, I use it on all my machines with extreme satisfaction and I will never thank you enough for it.

**Comment by MichaelEischer** [comment]
The fix for the issue still seems to be work in progress: https://github.com/golang/go/issues/54100

From what I can tell, https://github.com/golang/go/commit/b15c399a36a38509ae56dd69670974566f7b0d52 is not part of any released go version so far.

**Comment by MichaelEischer** [comment]
The bug is apparently fixed in FreeBSD 13.2, see #3550.
